apps/condo/domains/common/utils/apolloErrorFormatter.js

```diff
@@ -145,8 +147,15 @@ const formatError = error => {
     return safeFormatError(error, IS_HIDE_INTERNALS)
 }
 
+// NEW GraphQL Error standard
+
+function throwAuthenticationError () {
+    throw new AuthenticationError('No or incorrect authentication credentials')
```

Comment:
Just a question - how the mobile client is going to handle the errors? Is it OK for us to send them without translation? 

If this is ok and error is handled inside mobile app, then I think we should move these constants to some secure place, like errorConsts.js

Reply:
Yes, currently we don't have an error translation. At the moment we have an error name.
We will solve it in the future.